flixel/addons/nape/FlxNapeSpace.hx

```diff
@@ -45,7 +45,15 @@ class FlxNapeSpace extends FlxBasic
 	public static var drawDebug(default, set):Bool;
 	
 	#if !FLX_NO_DEBUG
-	private static var shapeDebug:ShapeDebug;
+	/**
+	 * A useful "canvas" or sprite-like object which can be used to draw debug information on, such as lines, circles and more.
+	 * To give an even better idea of its use, see the official Nape demo 'SpatialQueries' (http://napephys.com/samples.html#swf-SpatialQueries)
+	 * where this is used to draw lines emitted from Rays.
+	 * To use this in HaxeFlixel, place the code that uses shapeDebug in your FlxState's draw()-method, but make sure it only runs when FlxNapeSpace.drawDebug = true, as it
```

Comment:
The single `=` is strange here in the context of programming, should either be `==` or just `is` IMO.

`as it will crash the program if it's not.` is not very specific, could just mention that it's null in that case.